src/main/java/com/example/glovo/contoller/OrderController.java

```diff
@@ -0,0 +1,54 @@
+package com.example.glovo.contoller;
+
+import com.example.glovo.model.Order;
+import com.example.glovo.model.Product;
+import com.example.glovo.service.OrderService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Map;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+
+    private OrderService orderService;
```

Comment:
You have not injected service to the spring context, because currently you have only default constructor. Use annotation @AllArgsConstructor